apps/src/applab/applab.js

```diff
@@ -855,7 +853,7 @@ Applab.init = function (config) {
       localeDirection={studioApp.localeDirection()}
       editCode={!!config.level.editCode}
       readonlyWorkspace={Applab.reduxStore.getState().level.isReadOnlyWorkspace}
-      extraControlRows={extraControlRows}
+      showDebugger={showDebugButtons || showDebugConsole}
```

Comment:
I'm not sure one is better, but since we're already passing these properties into redux, I'd go ahead and let `JsDebugger` own this logic and render nothing if they're disabled, since it's already a connected component.

FYI, now that we're on React 15+ you can render `null` instead of an empty div.  [React will render it as an HTML comment.](https://facebook.github.io/react/blog/2016/04/07/react-v15.html#rendering-null-now-uses-comment-nodes)